subsys/bluetooth/mesh/Kconfig

```diff
@@ -292,6 +299,21 @@ config BT_MESH_TX_SEG_MAX
 	  which leaves 56 bytes for application layer data using a
 	  4-byte MIC and 52 bytes using an 8-byte MIC.
 
+config BT_MESH_TX_SEG_RETRANS_CNT
+	int "Number of retransmit attempts"
+	default 4
+	range 1 8
+	help
+	  Maximun number of retransmit attempts for outgoing segment 
+	  message.
+
+config BT_MESH_TX_SEG_RETRANS_TIMEOUT
+	int "Timeout value of retransmit segment message to group address"
```

Comment:
Same as above, e.g. "Transport message segment retransmit interval for group messages"